source/actors/player/Hero.hx

```diff
@@ -39,6 +39,15 @@ class Hero extends FlxSprite {
         standardMovement(moveDirection);
     }
 
+    public function getX()
+        {
+            return x;
+        }
+    public function getY()
+        {
+            return y;
+        }
+
```

Comment:
External objects should just be able to access the `x` and `y` properties of the sword directly. Was there a particular reason for using these getter methods?